### PR TITLE
WD-7718 - feat: Remove additional controllers feature

### PR DESCRIPTION
**Pull request by vladimir-cucu**
## Done

- Removed additional controllers feature.

## QA

- Verify that "Register a controller" button does not appear in controllers page.
- Verify that `/controllers?panel=register-controller` route doesn't open the register controller panel.
- Verify that artifacts related to additional controller feature have been removed.

## Details

- https://warthogs.atlassian.net/browse/WD-7718

## Screenshots

Screenshot from 2023-12-06 05-52-29

### CODE REVIEW

**Review by huwshimi — COMMENTED** [review]
Looks good. Just need to confirm this feature can be removed before we merge it.

**Comment by huwshimi** [comment]
@vladimir-cucu one other thing we should remove in this PR is the docs for adding additional controllers: https://github.com/canonical/juju-dashboard/blob/main/HACKING.md#adding-additional-controllers.

**Comment by vladimir-cucu** [comment]
> Looks good. Just need to confirm this feature can be removed before we merge it.

I converted the PR to draft, so that we do not merge it by mistake until the feature is confirmed.

> @vladimir-cucu one other thing we should remove in this PR is the docs for adding additional controllers: https://github.com/canonical/juju-dashboard/blob/main/HACKING.md#adding-additional-controllers.

Done! Thanks for the review.

**Review by huwshimi — APPROVED** [review]

**Pull request merged**
